modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -33,6 +33,7 @@ Optional API endpoint.
 
 - AWS: When blank, this is automatically generated using <<cloud_storage_region,region>> and <<cloud_storage_bucket,bucket>>. Otherwise, this uses the value assigned.
 - GCP: Uses `storage.googleapis.com`.
+- Azure: If you have enabled hierarchical namespaces for your storage account and use a custom domain, use <<cloud_storage_azure_adls_endpoint,`cloud_storage_azure_adls_endpoint`>>.
```

Comment:
Currently in our [Tiered Storage doc](https://docs.redpanda.com/current/manage/tiered-storage/?tab=tabs-1-microsoft-absadls#configure-object-storage) we have a note that says

> For hierarchical namespaces created with a custom endpoint, set cloud_storage_azure_adls_endpoint and cloud_storage_azure_adls_port. If you haven’t configured custom endpoints in Azure, there’s no need to edit these properties.

- Should **both** 1) HNS enabled and 2) custom endpoints set up be **True** in order to use `cloud_storage_azure_adls_endpoint` and `cloud_storage_azure_adls_port`? (Is it possible for users to have HNS enabled, but not have custom endpoints for their Azure storage account?)
- If Azure users do not have HNS enabled, do they then need to configure `cloud_storage_api_endpoint` (as with AWS/GCP users)?
- We specify that these `endpoint` properties are "Optional: No", but at the same time we say "Default: null" (and `cloud_storage_api_endpoint` is also [described](https://docs.redpanda.com/current/reference/properties/object-storage-properties/#cloud_storage_api_endpoint) as "Optional API endpoint"). This is confusing for readers. Can we clarify what config properties are required to use Azure for tiered storage vs what are optional, and in which scenarios?